utils/grafana.js

```diff
@@ -0,0 +1,81 @@
+import { parse as parseUrl, addParam } from '@/utils/url';
+
+export function computeDashboardUrl(embedUrl, clusterId, params) {
+  const url = parseUrl(embedUrl);
+  const clusterPrefix = clusterId === 'local' ? '' : `/k8s/clusters/${ clusterId }`;

```

Comment:
I'm kinda intrigued, do we not proxy k8s requests to the local cluster the same way?